.gitignore

```diff
@@ -88,3 +88,87 @@ lint/tmp/
+/app/src/main/res/drawable/ic_letter_g2.xml
+/app/src/main/res/drawable/ic_letter_h.xml
+/app/src/main/res/drawable/ic_letter_h2.xml
+/app/src/main/res/drawable/ic_letter_i.xml
+/app/src/main/res/drawable/ic_letter_i2.xml
+/app/src/main/res/drawable/ic_letter_j.xml
+/app/src/main/res/drawable/ic_letter_j2.xml
+/app/src/main/res/drawable/ic_letter_k.xml
+/app/src/main/res/drawable/ic_letter_l.xml
+/app/src/main/res/drawable/ic_letter_l2.xml
+/app/src/main/res/drawable/ic_letter_m.xml
+/app/src/main/res/drawable/ic_letter_m2.xml
+/app/src/main/res/drawable/ic_letter_n.xml
+/app/src/main/res/drawable/ic_letter_n2.xml
+/app/src/main/res/drawable/ic_letter_o.xml
+/app/src/main/res/drawable/ic_letter_o2.xml
+/app/src/main/res/drawable/ic_letter_p.xml
+/app/src/main/res/drawable/ic_letter_q.xml
+/app/src/main/res/drawable/ic_letter_q2.xml
+/app/src/main/res/drawable/ic_letter_r.xml
+/app/src/main/res/drawable/ic_letter_r2.xml
+/app/src/main/res/drawable/ic_letter_s.xml
+/app/src/main/res/drawable/ic_letter_t.xml
+/app/src/main/res/drawable/ic_letter_u.xml
+/app/src/main/res/drawable/ic_letter_u2.xml
+/app/src/main/res/drawable/ic_letter_v.xml
+/app/src/main/res/drawable/ic_letter_w.xml
+/app/src/main/res/drawable/ic_letter_w2.xml
+/app/src/main/res/drawable/ic_letter_x.xml
+/app/src/main/res/drawable/ic_letter_x2.xml
+/app/src/main/res/drawable/ic_letter_y.xml
+/app/src/main/res/drawable/ic_letter_z.xml
+/app/src/main/res/drawable/ic_letter_z2.xml
+/app/src/main/res/drawable/ic_number_0.xml
+/app/src/main/res/drawable/ic_number_1.xml
+/app/src/main/res/drawable/ic_number_10.xml
+/app/src/main/res/drawable/ic_number_11.xml
+/app/src/main/res/drawable/ic_number_12.xml
+/app/src/main/res/drawable/ic_number_13.xml
+/app/src/main/res/drawable/ic_number_14.xml
+/app/src/main/res/drawable/ic_number_15.xml
+/app/src/main/res/drawable/ic_number_16.xml
+/app/src/main/res/drawable/ic_number_17.xml
+/app/src/main/res/drawable/ic_number_18.xml
+/app/src/main/res/drawable/ic_number_19.xml
+/app/src/main/res/drawable/ic_number_2.xml
+/app/src/main/res/drawable/ic_number_20.xml
+/app/src/main/res/drawable/ic_number_3.xml
+/app/src/main/res/drawable/ic_number_4.xml
+/app/src/main/res/drawable/ic_number_5.xml
+/app/src/main/res/drawable/ic_number_6.xml
+/app/src/main/res/drawable/ic_number_7.xml
+/app/src/main/res/drawable/ic_number_8.xml
+/app/src/main/res/drawable/ic_number_9.xml
+/app/src/main/res/drawable/icnumber10.xml
+/app/src/main/res/drawable/icnumber11.xml
+/app/src/main/res/drawable/icnumber12.xml
+/app/src/main/res/drawable/icnumber13.xml
+/app/src/main/res/drawable/icnumber14.xml
+/app/src/main/res/drawable/icnumber15.xml
+/app/src/main/res/drawable/icnumber16.xml
+/app/src/main/res/drawable/icnumber17.xml
+/app/src/main/res/drawable/icnumber18.xml
+/app/src/main/res/drawable/icnumber19.xml
+/app/src/main/res/drawable/icnumber20.xml
+/app/src/main/res/drawable/icnumber8.xml
+/app/src/main/res/drawable/icnumber9.xml
```

Comment:
I'm not sure why these are in the gitignore file - could it be that these need to be committed instead?